config/locales/en.yml

```diff
@@ -991,6 +992,7 @@ en:
         goto_page_doesnt_exist: The question you’re taking the person to no longer exists - edit question %{page_index}’s route
     none_of_the_above: None of the above
     route: Route
+    secondary_skip_description: After %{check_page_text} go to %{goto_page_text}
```

Comment:
I think we want the question number of the check page in here as well, see https://app.mural.co/t/gaap0347/m/gaap0347/1728478914347/6a7e0b709f50f00f81cab37414fdf44ec8601a70?wid=0-1730911072926